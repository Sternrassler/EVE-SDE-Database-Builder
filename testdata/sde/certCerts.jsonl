{"certID":1,"description":"Basic gunnery certificate","groupID":266,"name":"Gunnery"}
{"certID":2,"description":"Basic missile certificate","groupID":267,"name":"Missile Operation"}
{"certID":3,"description":"Basic shield operation","groupID":268,"name":"Shield Operation"}
