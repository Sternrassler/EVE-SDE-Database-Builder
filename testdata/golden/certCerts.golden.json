[
  {
    "certificateID": 0,
    "description": "Basic gunnery certificate",
    "groupID": 266,
    "name": "Gunnery"
  },
  {
    "certificateID": 0,
    "description": "Basic missile certificate",
    "groupID": 267,
    "name": "Missile Operation"
  },
  {
    "certificateID": 0,
    "description": "Basic shield operation",
    "groupID": 268,
    "name": "Shield Operation"
  }
]